sdk/typescript/test/e2e/txn-serializer.test.ts

```diff
@@ -297,4 +298,74 @@ describe('Transaction Serialization and deserialization', () => {
     } as UnserializedSignableTransaction;
     expect(expectedTx).toEqual(deserialized);
   });
+
+  it('Check mutable reference is true', async () => {
+    const coins = await toolbox.provider.getGasObjectsOwnedByAddress(
+      toolbox.address(),
+    );
+    const validators = await toolbox.getActiveValidators();
+    const validator_metadata = (validators[0] as SuiMoveObject).fields.metadata;
+    const validator_address = (validator_metadata as SuiMoveObject).fields
+      .sui_address;
+    const moveCall = {
+      packageObjectId: '0x2',
+      module: 'sui_system',
+      function: 'request_add_delegation_mul_coin',
+      typeArguments: [],
+      arguments: [
+        SUI_SYSTEM_STATE_OBJECT_ID,
+        [coins[2].objectId],
+        [String(100)],
+        validator_address,
+      ],
+      gasBudget: DEFAULT_GAS_BUDGET,
+      gasPayment: coins[1].objectId,
+    } as MoveCallTransaction;
+
+    const localTxnBytes = await localSerializer.serializeToBytes(
+      toolbox.address(),
+      { kind: 'moveCall', data: moveCall },
+    );
```

Comment:
Since you also changed the Rust code, can you also test if the rpcSerializer also returns the correct result?